tsconfig.json

```diff
@@ -63,6 +63,6 @@
     /* Advanced Options */
     "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
   },
-  "include": ["**/*.ts"],
+  "include": ["scripts/*.ts", "migrations/*.ts"],
```

Comment:
Do I need to build src/*.ts for any external project?

Reply:
I am pretty sure `src/*.ts` is the only directory that needs to be built for external projects.

Reply:
Also, `migrations.ts` does not contain any TS.

Reply:
Yes, we should include `src/*.ts` as they need to be built and are included in the npm package.